layouts/black-lives-matter.hbs

```diff
@@ -220,9 +221,10 @@
       <li>William Green</li>
       <li>Willie McCoy</li>
       <li>Yassin Mohamed</li>
+      <li>Yehuda Biadga</li>
       <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
     </ul>
-    <p>We've listed the names of the Black folks who have been murdered by police brutality in the U.S. dating back to 2009, both because the U.S. is the origin of the Black Lives Matter movement and currently the heart of the resistance.</p>
+    <p>We've listed the names of the Black folks who have been murdered by police brutality in the U.S. and other places in the world dating back to 2009, both because the U.S. is the origin of the Black Lives Matter movement and currently the heart of the resistance.</p>
```

Comment:
This is going to conflict with https://github.com/nodejs/nodejs.org/pull/3212.  If we land that one first, I'd be happy to suggest a modification. (It probably just means adding something to the affect that instead of being an entirely U.S. list, it is a _predominantly_ U.S. list.

Reply:
@Trott I don't feel strongly about this PR landing at all (or attribution fwiw), I do think it makes sense to include these names and outline that this is a global issue the project is objecting to.